tox.ini

```
@@ -15,6 +15,10 @@ commands =
     pytest --pyargs glue_solar {posargs}
 
 [testenv:build_docs]
-changedir = {toxinidir}
-extras = docs
-commands = sphinx-build docs docs/_build/html -W -b html -d docs/_build/.doctrees
+description = invoke sphinx-build to build the HTML docs
+changedir = docs
+extras =
+  tests
+  qt
+  docs
+commands = sphinx-build -vvv --color -W --keep-going -b html -d _build/.doctrees . _build/html
```

```suggestion
commands = sphinx-build --color -W --keep-going -b html -d _build/.doctrees . _build/html
```
We don't want the full log from sphinx normally.

---

Maybe only `-v` is needed.